src/assets/scss/layouts/_forms.scss

```diff
@@ -0,0 +1,69 @@
+
+form{
+    margin-top: 20px;
+    
+    .field{
+        position: relative;
+        width: 100%;
+        margin-bottom: 20px;
+        label{
+            width: 100%;
+            position: relative;
+            color: $title_color;
+            display: inline-block;
+        }
+        
+        input[type=text],input[type=email], input[type=password], input[type=date], input[type=submit]{
+            border:none;
+            outline: none;
+            height: 40px;
+            margin-right:7px;
+            border: 1px solid $grey;
+            border-radius: 5px;
+            background:transparent;
+            padding-left:10px;
+            color:$dark;
+            font-family: $font-family, sans-serif;
+            &:focus{
+                border-bottom: 1px solid lighten($grey,50%);
+               
+            }
+            
+        }
+
+        input[type=submit]{
+            background-color: $title_color;
+            height: 50px;
+            color:$white;
+            border-radius: 10px;
+            font-size: 18px;
+        }
+        
+        input[type=email], #username,input[type=date],input[type=submit] {
+            width:95%;
+        }
+        input[type=text].title-input{
+            font-size: 27px;
+        }
+
+    }
+}
+#col-login{
+input[type=text], input[type=password], input[type=submit]{
+    width:92.3%;
+    height: 40px;   
+    }
+} 
+
+
+@media(max-width:768px){
```

Comment:
Prevent hardcoded media queries. To do that better, create a new partial, then declare device breakpoints to make them reusable. Maybe you can reach out to me to get help with this.